src/main/java/org/wildfly/security/auth/util/GSSCredentialSecurityFactory.java

```diff
@@ -393,11 +393,11 @@ private Configuration createConfiguration() throws IOException {
             if (IS_IBM) {
                 options.put("noAddress", "true");
                 options.put("credsType", (isServer && !obtainKerberosTicket) ? "acceptor" : "both");
-                options.put("useKeytab", keyTab.toURI().toURL().toString());
+                if (keyTab != null) options.put("useKeytab", keyTab.toURI().toURL().toString());
             } else {
                 options.put("storeKey", "true");
                 options.put("useKeyTab", "true");
```

Comment:
Should this still be true even if keyTab == null?

Reply:
To set useKeyTab=true without setting of keytab is valid:
> If keytab is not set then the module will locate the keytab from the Kerberos configuration file. If it is not specified in the Kerberos configuration file then it will look for the file {user.home}{file.separator}krb5.keytab.

https://docs.oracle.com/javase/7/docs/jre/api/security/jaas/spec/com/sun/security/auth/module/Krb5LoginModule.html

(even through we use undefined keyTab primary in combination with native kerberos, which ignores this properties at all)